app/vintage/tests/unittests/unittests/auth_helper.py

```diff
@@ -1,7 +1,7 @@
 from flask import url_for
 
-from app.helpers.data import DataManager, save_to_db
-from app.helpers.helpers import get_serializer
+from app.api.helpers.data import DataManager, save_to_db
```

Comment:
save_to_db I believe is already there in db.py helpers. So you can remove it from data.py.